### ISSUE TITLE
Basting Box?

### ISSUE THREAD

**Issue by HoneyPhantomhive** [inkstitch/inkstitch#440]
NOT AN ISSUE!!!

How would you make a basting box in the program like the one shown below? Thanks!

![image](https://user-images.githubusercontent.com/45114379/56327131-99571b00-613e-11e9-82d2-ca8b810d7864.png)

**Comment by goldfish** [comment]
Basting box is just a running stitch around the design. You create a rectangle/square around your design or to the size you want if you're adding it to another design. Set Fill to nothing. Then set the stroke style to a dashed line, it doesn't matter which one. When you go into Params it will do it as a running stitch instead of the default zig-zag at the stroke width.

**Comment by lexelby** [comment]
Don't forget to do Path -> Object to Path after using the rectangle tool :)

**Comment by tyrosinase** [comment]
Related: If anybody knows how to make a Brother PE-800 not do the tackdown stitches at start/stop, let me know.

**Comment by wwderw** [comment]
Are you talking about the tie-in, tie-off stitches?  If you are, you actually want that, otherwise you run the risk of being able to pull the thread out.

**Comment by lexelby** [comment]
I think, based on how my brother SE-400 behaves, that brother single-needles may add _extra_ tie-in stitches of their own, in addition to the ones that Ink/Stitch adds.  It's kind of a pain.

**Comment by wwderw** [comment]
I haven't observed that with my PRs, but I'll see if they do it as well.

Could always see about adding in a small stitch filter that would be user configurable and if a user makes it so small, it has been known to take out tie in/off stitches, but if Brother is adding them back in, might be a way of eliminating that.

May also look into the machines configuration settings to see if there is a way to turn it off there.

I've noticed some quirks of Brother files that have led me to do some work arounds compared to other machines.

**Comment by goldfish** [comment]
My Singer does something similar and it's their software that does it. It will add a lock stitch right in the middle of the hoop before it starts the current colour. Which is super annoying. You can delete them in the stitch plan if your software will let you do that. I usually just manually skip the first few stitches on the machine once the design is loaded though. It's almost habit and part of my workflow now.

**Comment by tyrosinase** [comment]
Yeah, it's the machine itself. If you pause in the middle of a run it'll do more to start itself off.

And yeah, 99% of the time you absolutely want it, but for basting stitches it'd be nice if it didn't happen, and also I am a big weirdo who does non-standard things with my machine like punch holes in paper for non-machine embroidery: https://silverseams.com/2019/03/06/embroidering-cards/

(I thread the machine except the needle itself, then hold the needle and it *generally* doesn't notice there's no thread. But occasionally it does, and then I get extraneous holes.

**Comment by wwderw** [comment]
> If you pause in the middle of a run it'll do more to start itself off.

That it shouldn't be doing.  It might start slower and pick up to the running speed, but it shouldn't be doing tie in stitches in the middle of the run after a pause (or not after a pause).   Now, after a trim or thread break or something like that in the middle of a run, yes, but not if it was just paused and then started back up again.

**Comment by lexelby** [comment]
I believe my SE400 does something similar.  These machines are considered entry-level, so it wouldn't surprise me if they did this just in case you had to switch thread spools or replace the bobbin or something.  Annoying.

@tyrosinase You may already know this, but if you enable "manual mode" on a running stitch object, Ink/Stitch will not add its own tie-in or tie-off for that object.  You can even embroider a normal running stitch using the Embroider extension and then copy and use the resulting object out of the stitch plan layer.  It will be already set up as a manual-mode running stitch.

**Comment by tyrosinase** [comment]
Yep! I went in and checked the stitch object to make sure it wasn't going to punch any extraneous holes, took it over to the machine, and was like DANG IT MACHINE STOP "HELPING."

And yeah, I think you're probably right about a normal pause, but the pauses I was getting was because the machine was unhappy about running without thread. So, not just pushing the big green button, but throwing an error on the screen, expecting that I had re-threaded the needle and therefore it needed to re-anchor the new thread, etc.

So ideally for the cards I can start and end the imaginary line of stitching outside of the paper, let it do the anchor on the stabilizer, then as long as I can successfully keep the end of the thread under tension *as if* it's in the needle, I can get it to punch only the holes in the paper that I have specifically told it to.

I'm kind of tempted to call Brother tech support and ask them how to turn the anchor feature off, just for the "you're trying to do WHAT with your embroidery machine?" and then the "but whyyyyyy."

Mother-in-law, who is the one who does card embroidery, has an Ellagio, so that's really the one whose behavior matters though.

**Comment by lexelby** [comment]
My SE400 seized up awhile back.  It was out of warranty so I bought the service manual from some shady site (I have it if anyone wants it) and took the machine apart.

Why am I mentioning this?  Because one thing I needed to do while unseizing it was to get it to run without thread so I could test it.

The top sensor is super-easy to trick, just hold or tape the spring up -- can probably be done with just the upper left cover off.  Maybe you could get super fancy and wire in a switch ;)

The bobbin sensor was a little tricker; I had to stick a piece of paper in there to block the light from going through.  The LED is just left of the bobbin case and the sensor is on the right.  I bet you could trick it just by putting masking tape on the center of an empty bobbin -- no need to disassemble anything.

Hope that helps :)

Oh, and I looked through the service manual, but it doesn't have anything about disabling tie stitches :(

**Comment by tyrosinase** [comment]
Thanks! It *is* super-easy to trick the top sensor; I have really only messed with it the one time, but I'm pretty sure if you just hold the thread tight it will do fine. I should have taped it in place because I just wasn't coordinated enough to remember to maintain tension whenever I leaned over to peek in and see what it was doing. I hadn't opened it, but I figured there was a simple spring there.

And my bobbin sensor has stopped sensing all on its own, which probably means I need to clean it out though again. Minky bits get EVERYWHERE. (My keyboard is a rainbow of them right now. Trimming stuffed animals in front of the screen is a bad habit.)
